install/kubernetes/quick-install.yaml

```diff
@@ -111,25 +144,37 @@ data:
   enable-bpf-masquerade: "true"
   enable-xt-socket-fallback: "true"
   install-iptables-rules: "true"
+  iptables-lock-timeout: "5s"
   auto-direct-node-routes: "false"
   enable-bandwidth-manager: "true"
+  enable-ipv4-fragment-tracking: "true"
   kube-proxy-replacement:  "probe"
+  node-port-range: "30000,32767"
+  node-port-mode: "snat"
   enable-health-check-nodeport: "true"
+  node-port-acceleration: "disabled"
   node-port-bind-protection: "true"
   enable-auto-protect-node-port-range: "true"
+  bpf-lb-maglev-table-size: "251"
```

Comment:
@brb The default for upgrades is `16381` and new installations is `251`. Which one should we pick?

Reply:
Why did we add the table size val to quick-install.yaml? Asking, as the default node port algorithm is `random` which doesn't use Maglev.

Reply:
removed this unless the nodeport algorithm is set to maglev